docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -0,0 +1,109 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Deploying on Google cloud platform
+
+include::./attributes.adoc[]
+
+This guide covers:
+
+* Connecting to a CloudSQL postgresql instance with quarkus-agroal and service account using CloudRun
+
+== Prerequisites
+
+For this guide you need:
+
+* roughly 1 hour
+* having access to an Google cloud platform project with owner rights.
+
+This guide will take as input an application developed in the link:datasource.adoc[datasource guide].
+
+Make sure you have the application at hand working locally.
+
+
+== Solution
+
+We recommend to follow the instructions in the next sections and build the application step by step.
+
+== Connecting to CloudSQL
+
+Google CloudSQL managed service allows 4 kinds of connection :
+
+. Using public IP
+. Using private IP
+. Using Cloud SQL Proxy
+. Using service account
+
+The first two don't need anymore details, simply connect to you database following the link:datasource.adoc[datasource guide].
+The third allows you to connect as if locally, but is not available for AppEngine or CloudRun where you only deploy one artefact.
+Please check link:https://cloud.google.com/sql/docs/postgres/external-connection-methods?hl=en[Connection options for external applications].
+
+This guide will help you through the fourth possibility : connecting using service account.
+
+== Adding necessary dependencies to your application.
+
+You will need to add the _Cloud SQL Postgres Socket Factory_ and postgresql driver dependencies.
+
+With maven :
+[source,xml, subs="attributes"]
+----
+<!-- https://mvnrepository.com/artifact/com.google.cloud.sql/postgres-socket-factory -->
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>postgres-socket-factory</artifactId>
+    <version>1.0.15</version>
+</dependency>
+<!-- https://mvnrepository.com/artifact/org.postgresql/postgresql -->
```

Comment:
Could we point to the more standard https://search.maven.org/ (or remove the links altogether)?